scripts/publish-module.gradle

```diff
@@ -0,0 +1,80 @@
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
+task androidSourcesJar(type: Jar) {
+    archiveClassifier.set('sources')
+    if (project.plugins.findPlugin("com.android.library")) {
+        from android.sourceSets.main.java.srcDirs
+    } else {
+        from sourceSets.main.java.srcDirs
+    }
+}
+
+task javadocJar(type: Jar) {
+    from javadoc
+    archiveClassifier.set('javadoc')
+}
+
+artifacts {
+    archives androidSourcesJar
+    archives javadocJar
+}
+group = PUBLISH_GROUP_ID
+version = PUBLISH_VERSION
+
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                groupId PUBLISH_GROUP_ID
+                artifactId PUBLISH_ARTIFACT_ID
+                version PUBLISH_VERSION
+                if (project.plugins.findPlugin("com.android.library")) {
+                    from components.release
+                } else {
+                    from components.java
+                }
+
+                artifact androidSourcesJar
+                artifact javadocJar
+
+                pom {
+                    name = PUBLISH_ARTIFACT_ID
+                    description = 'Astarte platform Java SDK'
+                    url = 'https://astarte-platform.org'
+                    licenses {
+                        license {
+                            name = 'Astarte platform Java SDK License'
+                            url = 'https://github.com/astarte-platform/astarte-device-sdk-java/blob/master/LICENSE'
+                        }
+                    }
+                    developers {
+                        developer {
+                            id = 'rbino'
+                            name = 'Riccardo Binetti'
+                            email = 'riccardo.binetti@seco.com'
```

Comment:
Riccardo main mail address is riccardo.binetti@ispirata.com, let's keep using it (since it is used everywhere).
We'll change it as soon there will be another official one.

Reply:
ok